docs/ide-usage.md

```diff
@@ -112,35 +105,21 @@ You can sign up for the free [JetBrains student license](https://www.jetbrains.c
     1. Go to `Build, Execution, Deployment → Build Tools → Gradle`. Under `Global Gradle settings`,
        add `-Dide=idea` to `Gradle VM options:`.
 
-1. You need to have an application server named `AppEngine Dev <version>` pointing to the SDK you downloaded previously as follows: 
-    
+1. You need the `Google Cloud Tools` plugin installed and configured:
+
+    1. If you have no current projects open, click `Configure → Plugins`.\
+       **OR**\
+       If you currently have projects open, click `File → Settings → Plugins` or `IntelliJ IDEA → Preferences → Plugins`.
+    1. Click on `Browse repositories...`. Select `Google Cloud Tools` and click `Install`.
+    1. During installation, you may encounter two prompts: one to install the `Google Account` plugin dependency and
+       another to disable the obsolete `Google App Engine Integration` plugin. Answer `Yes` to both prompts.
```

Comment:
Do we really need to mention the older plugin, though? It may only be relevant for a short while during transition.

Reply:
Yes, because it's still shipped with IntelliJ and enabled by default.